app/uk/gov/hmrc/exports/services/DeclarationService.scala

```diff
@@ -30,4 +30,8 @@ class DeclarationService @Inject()(declarationRepository: DeclarationRepository)
    */
   def save(declaration: ExportsDeclaration): Future[ExportsDeclaration] = declarationRepository.create(declaration)
 
+  def find(eori: String): Future[Seq[ExportsDeclaration]] = declarationRepository.find(eori)
```

Comment:
I am not sure which declaration would be returned when there is more then one submitted.

Reply:
Ok there is seq there so is should not be problem.

Reply:
Yeh I eventually see this superseding the "get by eori: endpoint